packages/autop/src/index.js

```diff
@@ -123,8 +123,8 @@ function replaceInHtmlTags( haystack, replacePairs ) {
  * replace double line-breaks with HTML paragraph tags. The remaining line-
  * breaks after conversion become `<br />` tags, unless br is set to 'false'.
  *
- * @param  {string}    text The text which has to be formatted.
- * @param  {boolean}   br   Optional. If set, will convert all remaining line-
+ * @param {string}  text The text which has to be formatted.
+ * @param {boolean} br   Optional. If set, will convert all remaining line-
  *                          breaks after paragraphing. Default true.
```

Comment:
```suggestion
 * @param {boolean} br   Optional. If set, will convert all remaining line-
 *                       breaks after paragraphing. Default true.
```

The original second line aligned with the start of the original description, seems like this should be updated here also